src/django_pg_migration_tools/management/commands/migrate_with_timeouts.py

```diff
@@ -69,3 +61,13 @@ def _migrate(self, *django_migrate_args: Any, **django_migrate_kwargs: Any) -> N
             statement_timeout=self.statement_timeout,
         ):
             super().handle(*django_migrate_args, **django_migrate_kwargs)
+
+
+def _parse_non_required_ms_option_to_timedelta(
+    options: Any, key: str
+) -> datetime.timedelta | None:
+    value = options.pop(key, None)
```

Comment:
🐼  The word "parse" in the function name suggests only parsing, not mutating the `options` dict. Naming things is hard, but perhaps "pop" or "extract" would be better.

Reply:
I've picked up the `pop` prefix instead.